### PR TITLE
Adding causal language

### PR DESCRIPTION
**Pull request by cetagostini**
## Description

Adding some causal language to the example notebook, showing how our model responds to a simple DAG, and making the assumptions within the structural equation built into the model even clearer.

## Related Issue

- [ ] Closes #7
- [x] Related to #724 

## Checklist

- [x] Checked that [the pre-commit linting/style checks pass](https://docs.pymc.io/en/latest/contributing/python_style.html)
- [ ] Included tests that prove the fix is effective or that the new feature works
- [x] Added necessary documentation (docstrings and/or example notebooks)
- [ ] If you are a pro: each commit corresponds to a [relevant logical change](https://wiki.openstack.org/wiki/GitCommitMessages#Structural_split_of_changes)

## Modules affected

- [x] MMM
- [ ] CLV

## Type of change

- [ ] New feature / enhancement
- [ ] Bug fix
- [x] Documentation
- [ ] Maintenance
- [ ] Other (please specify):

----
📚 Documentation preview 📚: https://pymc-marketing--1003.org.readthedocs.build/en/1003/

### CODE REVIEW

File: docs/source/notebooks/mmm/mmm_example.ipynb [hunk 1]
```diff
@@ -20,25 +20,101 @@
     "  - We also know that there is a yearly seasonality effect.\n",
```
Comment (drbenvincent) [inline docs/source/notebooks/mmm/mmm_example.ipynb:20]: <p>I don't like "causally compounded" because it sounds like "confounded". Maybe reword to emphasise "causal data generating process"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-labs/pymc-marketing/pull/1003/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (cetagostini) [reply docs/source/notebooks/mmm/mmm_example.ipynb:20]: Rephrase to `causally related.` sounds good? @drbenvincent

Comment (juanitorduz) [reply docs/source/notebooks/mmm/mmm_example.ipynb:20]: <p>Add explicitly what DAG means, i.e.</p><p> </p><p>"we can draw a directed acyclic graph (DAG) or a graphical model"</p>

Comment (juanitorduz) [inline docs/source/notebooks/mmm/mmm_example.ipynb:20]: <p>PyMC-Marketing (capital letters)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-labs/pymc-marketing/pull/1003/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

**Comment by juanitorduz** [comment]
Nice! I left some minor suggestions!

**Comment by cetagostini** [comment]
@juanitorduz Thanks for the review, I already modify the text based on it!

**Review by juanitorduz — APPROVED** [review]
Thanks @cetagostini !
